src/sdk/pynni/tests/test_pruners.py

```diff
@@ -34,7 +34,7 @@ def validate_sparsity(wrapper, sparsity, bias=False):
     'agp': {
         'pruner_class': AGP_Pruner,
         'config_list': [{
-            'initial_sparsity': 0,
+            'initial_sparsity': 0.,
```

Comment:
why use `0.` ?

Reply:
we require the sparsity field to be a float.